test/41-device-api-v1.spec.ts

```diff
@@ -615,5 +623,44 @@ describe('SupervisorAPI [V1 Endpoints]', () => {
 		});
 	});
 
+	describe('POST /v1/blink', () => {
+		// Further blink function-specific testing located in 07-blink.spec.ts
+		it('responds with code 200 and empty body', async () => {
+			await request
+				.post('/v1/blink')
+				.set('Accept', 'application/json')
+				.set('Authorization', `Bearer ${apiKeys.cloudApiKey}`)
+				.expect(sampleResponses.V1.POST['/blink'].statusCode)
+				.then((response) => {
+					expect(response.body).to.deep.equal(
+						sampleResponses.V1.POST['/blink'].body,
```

Comment:
This test won't run without the responses from the other commit right? I would recommend squashing both commits so you get a single passing change,

Reply:
Done!